misc/python/materialize/checks/large_tables.py

```diff
@@ -56,6 +56,7 @@ def initialize(self) -> Testdrive:
                 """
                 > CREATE TABLE many_rows (f1 TEXT);
                 > CREATE DEFAULT INDEX ON many_rows;
+                > SET statement_timeout = '120s';
```

Comment:
What is the default timeout? Is this a database or test setting? Database, I guess?

Reply:
Database setting, 10 s by default.